hack/structprotogen/consts/consts.go

```diff
@@ -237,7 +238,9 @@ func (b *ConstBlocks) FormatProtoFile(w io.Writer) error {
 
 		fmt.Fprintf(w, "enum %s {\n", block.ProtoMessageName())
 
-		if hasDuplicates(block.Consts, func(c Constant) string { return c.Value }) {
+		hasZeroNotFirstConstValue := slices.IndexFunc(block.Consts, func(c Constant) bool { return c.Value == "0" }) > 0
```

Comment:
this fixes a case when enum is like `{-100, 0, 100}`, so it has zero, but not as the first value